[ WARN ] [2016-03-23 12:11:34] io.netty.channel.AbstractChannel.Thread-2 [146] - Force-closing a channel whose registration task was not accepted by an event loop: [id: 0xa1b0b2fc]
java.util.concurrent.RejectedExecutionException: event executor terminated
	at io.netty.util.concurrent.SingleThreadEventExecutor.reject(SingleThreadEventExecutor.java:715)
	at io.netty.util.concurrent.SingleThreadEventExecutor.addTask(SingleThreadEventExecutor.java:300)
	at io.netty.util.concurrent.SingleThreadEventExecutor.execute(SingleThreadEventExecutor.java:691)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register(AbstractChannel.java:421)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:60)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:48)
	at io.netty.channel.MultithreadEventLoopGroup.register(MultithreadEventLoopGroup.java:64)
	at io.netty.bootstrap.AbstractBootstrap.initAndRegister(AbstractBootstrap.java:316)
	at io.netty.bootstrap.Bootstrap.doConnect(Bootstrap.java:135)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:117)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:98)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:90)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.run(DBPClient.java:84)
[ WARN ] [2016-03-23 12:11:39] io.netty.channel.AbstractChannel.Thread-2 [146] - Force-closing a channel whose registration task was not accepted by an event loop: [id: 0x5d193aeb]
java.util.concurrent.RejectedExecutionException: event executor terminated
	at io.netty.util.concurrent.SingleThreadEventExecutor.reject(SingleThreadEventExecutor.java:715)
	at io.netty.util.concurrent.SingleThreadEventExecutor.addTask(SingleThreadEventExecutor.java:300)
	at io.netty.util.concurrent.SingleThreadEventExecutor.execute(SingleThreadEventExecutor.java:691)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register(AbstractChannel.java:421)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:60)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:48)
	at io.netty.channel.MultithreadEventLoopGroup.register(MultithreadEventLoopGroup.java:64)
	at io.netty.bootstrap.AbstractBootstrap.initAndRegister(AbstractBootstrap.java:316)
	at io.netty.bootstrap.Bootstrap.doConnect(Bootstrap.java:135)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:117)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:98)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:90)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.run(DBPClient.java:84)
[ WARN ] [2016-03-23 12:11:44] io.netty.channel.AbstractChannel.Thread-2 [146] - Force-closing a channel whose registration task was not accepted by an event loop: [id: 0x164c88e6]
java.util.concurrent.RejectedExecutionException: event executor terminated
	at io.netty.util.concurrent.SingleThreadEventExecutor.reject(SingleThreadEventExecutor.java:715)
	at io.netty.util.concurrent.SingleThreadEventExecutor.addTask(SingleThreadEventExecutor.java:300)
	at io.netty.util.concurrent.SingleThreadEventExecutor.execute(SingleThreadEventExecutor.java:691)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register(AbstractChannel.java:421)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:60)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:48)
	at io.netty.channel.MultithreadEventLoopGroup.register(MultithreadEventLoopGroup.java:64)
	at io.netty.bootstrap.AbstractBootstrap.initAndRegister(AbstractBootstrap.java:316)
	at io.netty.bootstrap.Bootstrap.doConnect(Bootstrap.java:135)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:117)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:98)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:90)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.run(DBPClient.java:84)
[ WARN ] [2016-03-23 12:11:49] io.netty.channel.AbstractChannel.Thread-2 [146] - Force-closing a channel whose registration task was not accepted by an event loop: [id: 0x372c70b7]
java.util.concurrent.RejectedExecutionException: event executor terminated
	at io.netty.util.concurrent.SingleThreadEventExecutor.reject(SingleThreadEventExecutor.java:715)
	at io.netty.util.concurrent.SingleThreadEventExecutor.addTask(SingleThreadEventExecutor.java:300)
	at io.netty.util.concurrent.SingleThreadEventExecutor.execute(SingleThreadEventExecutor.java:691)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register(AbstractChannel.java:421)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:60)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:48)
	at io.netty.channel.MultithreadEventLoopGroup.register(MultithreadEventLoopGroup.java:64)
	at io.netty.bootstrap.AbstractBootstrap.initAndRegister(AbstractBootstrap.java:316)
	at io.netty.bootstrap.Bootstrap.doConnect(Bootstrap.java:135)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:117)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:98)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:90)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.run(DBPClient.java:84)
[ WARN ] [2016-03-23 12:11:49] com.tiza.util.client.impl.DBPClient.Thread-2 [100] - DBP客户端重连失败！
[ WARN ] [2016-03-23 12:13:09] io.netty.channel.AbstractChannel.Thread-2 [146] - Force-closing a channel whose registration task was not accepted by an event loop: [id: 0xc045ff64]
java.util.concurrent.RejectedExecutionException: event executor terminated
	at io.netty.util.concurrent.SingleThreadEventExecutor.reject(SingleThreadEventExecutor.java:715)
	at io.netty.util.concurrent.SingleThreadEventExecutor.addTask(SingleThreadEventExecutor.java:300)
	at io.netty.util.concurrent.SingleThreadEventExecutor.execute(SingleThreadEventExecutor.java:691)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register(AbstractChannel.java:421)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:60)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:48)
	at io.netty.channel.MultithreadEventLoopGroup.register(MultithreadEventLoopGroup.java:64)
	at io.netty.bootstrap.AbstractBootstrap.initAndRegister(AbstractBootstrap.java:316)
	at io.netty.bootstrap.Bootstrap.doConnect(Bootstrap.java:135)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:117)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:98)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:90)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.run(DBPClient.java:84)
[ WARN ] [2016-03-23 12:13:19] io.netty.channel.AbstractChannel.Thread-2 [146] - Force-closing a channel whose registration task was not accepted by an event loop: [id: 0xfebd087d]
java.util.concurrent.RejectedExecutionException: event executor terminated
	at io.netty.util.concurrent.SingleThreadEventExecutor.reject(SingleThreadEventExecutor.java:715)
	at io.netty.util.concurrent.SingleThreadEventExecutor.addTask(SingleThreadEventExecutor.java:300)
	at io.netty.util.concurrent.SingleThreadEventExecutor.execute(SingleThreadEventExecutor.java:691)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register(AbstractChannel.java:421)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:60)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:48)
	at io.netty.channel.MultithreadEventLoopGroup.register(MultithreadEventLoopGroup.java:64)
	at io.netty.bootstrap.AbstractBootstrap.initAndRegister(AbstractBootstrap.java:316)
	at io.netty.bootstrap.Bootstrap.doConnect(Bootstrap.java:135)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:117)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:98)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:90)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.run(DBPClient.java:84)
[ WARN ] [2016-03-23 12:13:29] io.netty.channel.AbstractChannel.Thread-2 [146] - Force-closing a channel whose registration task was not accepted by an event loop: [id: 0xb3904420]
java.util.concurrent.RejectedExecutionException: event executor terminated
	at io.netty.util.concurrent.SingleThreadEventExecutor.reject(SingleThreadEventExecutor.java:715)
	at io.netty.util.concurrent.SingleThreadEventExecutor.addTask(SingleThreadEventExecutor.java:300)
	at io.netty.util.concurrent.SingleThreadEventExecutor.execute(SingleThreadEventExecutor.java:691)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register(AbstractChannel.java:421)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:60)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:48)
	at io.netty.channel.MultithreadEventLoopGroup.register(MultithreadEventLoopGroup.java:64)
	at io.netty.bootstrap.AbstractBootstrap.initAndRegister(AbstractBootstrap.java:316)
	at io.netty.bootstrap.Bootstrap.doConnect(Bootstrap.java:135)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:117)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:98)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:90)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.run(DBPClient.java:84)
[ WARN ] [2016-03-23 12:13:39] io.netty.channel.AbstractChannel.Thread-2 [146] - Force-closing a channel whose registration task was not accepted by an event loop: [id: 0xc8e3ec80]
java.util.concurrent.RejectedExecutionException: event executor terminated
	at io.netty.util.concurrent.SingleThreadEventExecutor.reject(SingleThreadEventExecutor.java:715)
	at io.netty.util.concurrent.SingleThreadEventExecutor.addTask(SingleThreadEventExecutor.java:300)
	at io.netty.util.concurrent.SingleThreadEventExecutor.execute(SingleThreadEventExecutor.java:691)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register(AbstractChannel.java:421)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:60)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:48)
	at io.netty.channel.MultithreadEventLoopGroup.register(MultithreadEventLoopGroup.java:64)
	at io.netty.bootstrap.AbstractBootstrap.initAndRegister(AbstractBootstrap.java:316)
	at io.netty.bootstrap.Bootstrap.doConnect(Bootstrap.java:135)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:117)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:98)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:90)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.run(DBPClient.java:84)
[ WARN ] [2016-03-23 12:13:39] com.tiza.util.client.impl.DBPClient.Thread-2 [100] - DBP客户端重连失败！
[ WARN ] [2016-03-23 12:15:01] io.netty.channel.AbstractChannel.Thread-2 [146] - Force-closing a channel whose registration task was not accepted by an event loop: [id: 0x67c80a06]
java.util.concurrent.RejectedExecutionException: event executor terminated
	at io.netty.util.concurrent.SingleThreadEventExecutor.reject(SingleThreadEventExecutor.java:715)
	at io.netty.util.concurrent.SingleThreadEventExecutor.addTask(SingleThreadEventExecutor.java:300)
	at io.netty.util.concurrent.SingleThreadEventExecutor.execute(SingleThreadEventExecutor.java:691)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register(AbstractChannel.java:421)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:60)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:48)
	at io.netty.channel.MultithreadEventLoopGroup.register(MultithreadEventLoopGroup.java:64)
	at io.netty.bootstrap.AbstractBootstrap.initAndRegister(AbstractBootstrap.java:316)
	at io.netty.bootstrap.Bootstrap.doConnect(Bootstrap.java:135)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:117)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:98)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:90)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.run(DBPClient.java:84)
[ WARN ] [2016-03-23 12:15:11] io.netty.channel.AbstractChannel.Thread-2 [146] - Force-closing a channel whose registration task was not accepted by an event loop: [id: 0x18ef0e08]
java.util.concurrent.RejectedExecutionException: event executor terminated
	at io.netty.util.concurrent.SingleThreadEventExecutor.reject(SingleThreadEventExecutor.java:715)
	at io.netty.util.concurrent.SingleThreadEventExecutor.addTask(SingleThreadEventExecutor.java:300)
	at io.netty.util.concurrent.SingleThreadEventExecutor.execute(SingleThreadEventExecutor.java:691)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register(AbstractChannel.java:421)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:60)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:48)
	at io.netty.channel.MultithreadEventLoopGroup.register(MultithreadEventLoopGroup.java:64)
	at io.netty.bootstrap.AbstractBootstrap.initAndRegister(AbstractBootstrap.java:316)
	at io.netty.bootstrap.Bootstrap.doConnect(Bootstrap.java:135)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:117)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:98)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:90)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.run(DBPClient.java:84)
[ WARN ] [2016-03-23 12:15:21] io.netty.channel.AbstractChannel.Thread-2 [146] - Force-closing a channel whose registration task was not accepted by an event loop: [id: 0x7f3292a5]
java.util.concurrent.RejectedExecutionException: event executor terminated
	at io.netty.util.concurrent.SingleThreadEventExecutor.reject(SingleThreadEventExecutor.java:715)
	at io.netty.util.concurrent.SingleThreadEventExecutor.addTask(SingleThreadEventExecutor.java:300)
	at io.netty.util.concurrent.SingleThreadEventExecutor.execute(SingleThreadEventExecutor.java:691)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register(AbstractChannel.java:421)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:60)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:48)
	at io.netty.channel.MultithreadEventLoopGroup.register(MultithreadEventLoopGroup.java:64)
	at io.netty.bootstrap.AbstractBootstrap.initAndRegister(AbstractBootstrap.java:316)
	at io.netty.bootstrap.Bootstrap.doConnect(Bootstrap.java:135)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:117)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:98)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:90)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.run(DBPClient.java:84)
[ WARN ] [2016-03-23 12:15:31] io.netty.channel.AbstractChannel.Thread-2 [146] - Force-closing a channel whose registration task was not accepted by an event loop: [id: 0xc1fac6ea]
java.util.concurrent.RejectedExecutionException: event executor terminated
	at io.netty.util.concurrent.SingleThreadEventExecutor.reject(SingleThreadEventExecutor.java:715)
	at io.netty.util.concurrent.SingleThreadEventExecutor.addTask(SingleThreadEventExecutor.java:300)
	at io.netty.util.concurrent.SingleThreadEventExecutor.execute(SingleThreadEventExecutor.java:691)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register(AbstractChannel.java:421)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:60)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:48)
	at io.netty.channel.MultithreadEventLoopGroup.register(MultithreadEventLoopGroup.java:64)
	at io.netty.bootstrap.AbstractBootstrap.initAndRegister(AbstractBootstrap.java:316)
	at io.netty.bootstrap.Bootstrap.doConnect(Bootstrap.java:135)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:117)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:98)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:90)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.run(DBPClient.java:84)
[ WARN ] [2016-03-23 12:15:41] io.netty.channel.AbstractChannel.Thread-2 [146] - Force-closing a channel whose registration task was not accepted by an event loop: [id: 0xab279d7a]
java.util.concurrent.RejectedExecutionException: event executor terminated
	at io.netty.util.concurrent.SingleThreadEventExecutor.reject(SingleThreadEventExecutor.java:715)
	at io.netty.util.concurrent.SingleThreadEventExecutor.addTask(SingleThreadEventExecutor.java:300)
	at io.netty.util.concurrent.SingleThreadEventExecutor.execute(SingleThreadEventExecutor.java:691)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register(AbstractChannel.java:421)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:60)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:48)
	at io.netty.channel.MultithreadEventLoopGroup.register(MultithreadEventLoopGroup.java:64)
	at io.netty.bootstrap.AbstractBootstrap.initAndRegister(AbstractBootstrap.java:316)
	at io.netty.bootstrap.Bootstrap.doConnect(Bootstrap.java:135)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:117)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:98)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:90)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.run(DBPClient.java:84)
[ WARN ] [2016-03-23 12:15:41] com.tiza.util.client.impl.DBPClient.Thread-2 [100] - DBP客户端重连失败！
[ WARN ] [2016-03-23 12:16:36] io.netty.channel.AbstractChannel.Thread-2 [146] - Force-closing a channel whose registration task was not accepted by an event loop: [id: 0x318eacbe]
java.util.concurrent.RejectedExecutionException: event executor terminated
	at io.netty.util.concurrent.SingleThreadEventExecutor.reject(SingleThreadEventExecutor.java:715)
	at io.netty.util.concurrent.SingleThreadEventExecutor.addTask(SingleThreadEventExecutor.java:300)
	at io.netty.util.concurrent.SingleThreadEventExecutor.execute(SingleThreadEventExecutor.java:691)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register(AbstractChannel.java:421)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:60)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:48)
	at io.netty.channel.MultithreadEventLoopGroup.register(MultithreadEventLoopGroup.java:64)
	at io.netty.bootstrap.AbstractBootstrap.initAndRegister(AbstractBootstrap.java:316)
	at io.netty.bootstrap.Bootstrap.doConnect(Bootstrap.java:135)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:117)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:98)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:90)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.run(DBPClient.java:84)
[ WARN ] [2016-03-23 12:16:47] io.netty.channel.AbstractChannel.Thread-2 [146] - Force-closing a channel whose registration task was not accepted by an event loop: [id: 0x1a5a81c8]
java.util.concurrent.RejectedExecutionException: event executor terminated
	at io.netty.util.concurrent.SingleThreadEventExecutor.reject(SingleThreadEventExecutor.java:715)
	at io.netty.util.concurrent.SingleThreadEventExecutor.addTask(SingleThreadEventExecutor.java:300)
	at io.netty.util.concurrent.SingleThreadEventExecutor.execute(SingleThreadEventExecutor.java:691)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register(AbstractChannel.java:421)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:60)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:48)
	at io.netty.channel.MultithreadEventLoopGroup.register(MultithreadEventLoopGroup.java:64)
	at io.netty.bootstrap.AbstractBootstrap.initAndRegister(AbstractBootstrap.java:316)
	at io.netty.bootstrap.Bootstrap.doConnect(Bootstrap.java:135)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:117)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:98)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:90)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.run(DBPClient.java:84)
[ WARN ] [2016-03-23 12:16:57] io.netty.channel.AbstractChannel.Thread-2 [146] - Force-closing a channel whose registration task was not accepted by an event loop: [id: 0x67fefcfc]
java.util.concurrent.RejectedExecutionException: event executor terminated
	at io.netty.util.concurrent.SingleThreadEventExecutor.reject(SingleThreadEventExecutor.java:715)
	at io.netty.util.concurrent.SingleThreadEventExecutor.addTask(SingleThreadEventExecutor.java:300)
	at io.netty.util.concurrent.SingleThreadEventExecutor.execute(SingleThreadEventExecutor.java:691)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register(AbstractChannel.java:421)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:60)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:48)
	at io.netty.channel.MultithreadEventLoopGroup.register(MultithreadEventLoopGroup.java:64)
	at io.netty.bootstrap.AbstractBootstrap.initAndRegister(AbstractBootstrap.java:316)
	at io.netty.bootstrap.Bootstrap.doConnect(Bootstrap.java:135)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:117)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:98)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:90)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.run(DBPClient.java:84)
[ WARN ] [2016-03-23 12:17:07] io.netty.channel.AbstractChannel.Thread-2 [146] - Force-closing a channel whose registration task was not accepted by an event loop: [id: 0x60a3c2d4]
java.util.concurrent.RejectedExecutionException: event executor terminated
	at io.netty.util.concurrent.SingleThreadEventExecutor.reject(SingleThreadEventExecutor.java:715)
	at io.netty.util.concurrent.SingleThreadEventExecutor.addTask(SingleThreadEventExecutor.java:300)
	at io.netty.util.concurrent.SingleThreadEventExecutor.execute(SingleThreadEventExecutor.java:691)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register(AbstractChannel.java:421)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:60)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:48)
	at io.netty.channel.MultithreadEventLoopGroup.register(MultithreadEventLoopGroup.java:64)
	at io.netty.bootstrap.AbstractBootstrap.initAndRegister(AbstractBootstrap.java:316)
	at io.netty.bootstrap.Bootstrap.doConnect(Bootstrap.java:135)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:117)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:98)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:90)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.run(DBPClient.java:84)
[ WARN ] [2016-03-23 12:17:17] io.netty.channel.AbstractChannel.Thread-2 [146] - Force-closing a channel whose registration task was not accepted by an event loop: [id: 0xb3da8613]
java.util.concurrent.RejectedExecutionException: event executor terminated
	at io.netty.util.concurrent.SingleThreadEventExecutor.reject(SingleThreadEventExecutor.java:715)
	at io.netty.util.concurrent.SingleThreadEventExecutor.addTask(SingleThreadEventExecutor.java:300)
	at io.netty.util.concurrent.SingleThreadEventExecutor.execute(SingleThreadEventExecutor.java:691)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register(AbstractChannel.java:421)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:60)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:48)
	at io.netty.channel.MultithreadEventLoopGroup.register(MultithreadEventLoopGroup.java:64)
	at io.netty.bootstrap.AbstractBootstrap.initAndRegister(AbstractBootstrap.java:316)
	at io.netty.bootstrap.Bootstrap.doConnect(Bootstrap.java:135)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:117)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:98)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:90)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.run(DBPClient.java:84)
[ WARN ] [2016-03-23 12:17:27] io.netty.channel.AbstractChannel.Thread-2 [146] - Force-closing a channel whose registration task was not accepted by an event loop: [id: 0x06a1267b]
java.util.concurrent.RejectedExecutionException: event executor terminated
	at io.netty.util.concurrent.SingleThreadEventExecutor.reject(SingleThreadEventExecutor.java:715)
	at io.netty.util.concurrent.SingleThreadEventExecutor.addTask(SingleThreadEventExecutor.java:300)
	at io.netty.util.concurrent.SingleThreadEventExecutor.execute(SingleThreadEventExecutor.java:691)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register(AbstractChannel.java:421)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:60)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:48)
	at io.netty.channel.MultithreadEventLoopGroup.register(MultithreadEventLoopGroup.java:64)
	at io.netty.bootstrap.AbstractBootstrap.initAndRegister(AbstractBootstrap.java:316)
	at io.netty.bootstrap.Bootstrap.doConnect(Bootstrap.java:135)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:117)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:98)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:90)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.run(DBPClient.java:84)
[ WARN ] [2016-03-23 12:17:37] io.netty.channel.AbstractChannel.Thread-2 [146] - Force-closing a channel whose registration task was not accepted by an event loop: [id: 0x626f6bf4]
java.util.concurrent.RejectedExecutionException: event executor terminated
	at io.netty.util.concurrent.SingleThreadEventExecutor.reject(SingleThreadEventExecutor.java:715)
	at io.netty.util.concurrent.SingleThreadEventExecutor.addTask(SingleThreadEventExecutor.java:300)
	at io.netty.util.concurrent.SingleThreadEventExecutor.execute(SingleThreadEventExecutor.java:691)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register(AbstractChannel.java:421)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:60)
	at io.netty.channel.SingleThreadEventLoop.register(SingleThreadEventLoop.java:48)
	at io.netty.channel.MultithreadEventLoopGroup.register(MultithreadEventLoopGroup.java:64)
	at io.netty.bootstrap.AbstractBootstrap.initAndRegister(AbstractBootstrap.java:316)
	at io.netty.bootstrap.Bootstrap.doConnect(Bootstrap.java:135)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:117)
	at io.netty.bootstrap.Bootstrap.connect(Bootstrap.java:98)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:90)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.connectDBP(DBPClient.java:112)
	at com.tiza.util.client.impl.DBPClient.run(DBPClient.java:84)
[ WARN ] [2016-03-23 12:21:06] io.netty.channel.ChannelInitializer.nioEventLoopGroup-4-1 [151] - Failed to initialize a channel. Closing: [id: 0x97529a62]
io.netty.channel.ChannelPipelineException: com.tiza.util.client.handler.DBPHandler is not a @Sharable handler, so can't be added or removed multiple times.
	at io.netty.channel.DefaultChannelPipeline.checkMultiplicity(DefaultChannelPipeline.java:449)
	at io.netty.channel.DefaultChannelPipeline.addLast0(DefaultChannelPipeline.java:146)
	at io.netty.channel.DefaultChannelPipeline.addLast(DefaultChannelPipeline.java:141)
	at io.netty.channel.DefaultChannelPipeline.addLast(DefaultChannelPipeline.java:253)
	at io.netty.channel.DefaultChannelPipeline.addLast(DefaultChannelPipeline.java:240)
	at com.tiza.util.client.impl.DBPClient$1.initChannel(DBPClient.java:84)
	at com.tiza.util.client.impl.DBPClient$1.initChannel(DBPClient.java:76)
	at io.netty.channel.ChannelInitializer.channelRegistered(ChannelInitializer.java:68)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRegistered(AbstractChannelHandlerContext.java:133)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRegistered(AbstractChannelHandlerContext.java:119)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRegistered(DefaultChannelPipeline.java:725)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:450)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.access$100(AbstractChannel.java:378)
	at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:424)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:358)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:357)
	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:112)
	at io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:137)
	at java.lang.Thread.run(Thread.java:745)
[ WARN ] [2016-03-23 12:21:17] io.netty.channel.ChannelInitializer.nioEventLoopGroup-5-1 [151] - Failed to initialize a channel. Closing: [id: 0xd212a5e9]
io.netty.channel.ChannelPipelineException: com.tiza.util.client.handler.DBPHandler is not a @Sharable handler, so can't be added or removed multiple times.
	at io.netty.channel.DefaultChannelPipeline.checkMultiplicity(DefaultChannelPipeline.java:449)
	at io.netty.channel.DefaultChannelPipeline.addLast0(DefaultChannelPipeline.java:146)
	at io.netty.channel.DefaultChannelPipeline.addLast(DefaultChannelPipeline.java:141)
	at io.netty.channel.DefaultChannelPipeline.addLast(DefaultChannelPipeline.java:253)
	at io.netty.channel.DefaultChannelPipeline.addLast(DefaultChannelPipeline.java:240)
	at com.tiza.util.client.impl.DBPClient$1.initChannel(DBPClient.java:84)
	at com.tiza.util.client.impl.DBPClient$1.initChannel(DBPClient.java:76)
	at io.netty.channel.ChannelInitializer.channelRegistered(ChannelInitializer.java:68)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRegistered(AbstractChannelHandlerContext.java:133)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRegistered(AbstractChannelHandlerContext.java:119)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRegistered(DefaultChannelPipeline.java:725)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:450)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.access$100(AbstractChannel.java:378)
	at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:424)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:358)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:357)
	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:112)
	at io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:137)
	at java.lang.Thread.run(Thread.java:745)
[ WARN ] [2016-03-23 12:21:27] io.netty.channel.ChannelInitializer.nioEventLoopGroup-6-1 [151] - Failed to initialize a channel. Closing: [id: 0xab96b158]
io.netty.channel.ChannelPipelineException: com.tiza.util.client.handler.DBPHandler is not a @Sharable handler, so can't be added or removed multiple times.
	at io.netty.channel.DefaultChannelPipeline.checkMultiplicity(DefaultChannelPipeline.java:449)
	at io.netty.channel.DefaultChannelPipeline.addLast0(DefaultChannelPipeline.java:146)
	at io.netty.channel.DefaultChannelPipeline.addLast(DefaultChannelPipeline.java:141)
	at io.netty.channel.DefaultChannelPipeline.addLast(DefaultChannelPipeline.java:253)
	at io.netty.channel.DefaultChannelPipeline.addLast(DefaultChannelPipeline.java:240)
	at com.tiza.util.client.impl.DBPClient$1.initChannel(DBPClient.java:84)
	at com.tiza.util.client.impl.DBPClient$1.initChannel(DBPClient.java:76)
	at io.netty.channel.ChannelInitializer.channelRegistered(ChannelInitializer.java:68)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRegistered(AbstractChannelHandlerContext.java:133)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRegistered(AbstractChannelHandlerContext.java:119)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRegistered(DefaultChannelPipeline.java:725)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:450)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.access$100(AbstractChannel.java:378)
	at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:424)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:358)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:357)
	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:112)
	at io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:137)
	at java.lang.Thread.run(Thread.java:745)
[ WARN ] [2016-03-23 12:21:37] io.netty.channel.ChannelInitializer.nioEventLoopGroup-7-1 [151] - Failed to initialize a channel. Closing: [id: 0x6acda1c3]
io.netty.channel.ChannelPipelineException: com.tiza.util.client.handler.DBPHandler is not a @Sharable handler, so can't be added or removed multiple times.
	at io.netty.channel.DefaultChannelPipeline.checkMultiplicity(DefaultChannelPipeline.java:449)
	at io.netty.channel.DefaultChannelPipeline.addLast0(DefaultChannelPipeline.java:146)
	at io.netty.channel.DefaultChannelPipeline.addLast(DefaultChannelPipeline.java:141)
	at io.netty.channel.DefaultChannelPipeline.addLast(DefaultChannelPipeline.java:253)
	at io.netty.channel.DefaultChannelPipeline.addLast(DefaultChannelPipeline.java:240)
	at com.tiza.util.client.impl.DBPClient$1.initChannel(DBPClient.java:84)
	at com.tiza.util.client.impl.DBPClient$1.initChannel(DBPClient.java:76)
	at io.netty.channel.ChannelInitializer.channelRegistered(ChannelInitializer.java:68)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRegistered(AbstractChannelHandlerContext.java:133)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRegistered(AbstractChannelHandlerContext.java:119)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRegistered(DefaultChannelPipeline.java:725)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:450)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.access$100(AbstractChannel.java:378)
	at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:424)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:358)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:357)
	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:112)
	at io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:137)
	at java.lang.Thread.run(Thread.java:745)
[ WARN ] [2016-03-23 12:21:47] io.netty.channel.ChannelInitializer.nioEventLoopGroup-8-1 [151] - Failed to initialize a channel. Closing: [id: 0x93c45de0]
io.netty.channel.ChannelPipelineException: com.tiza.util.client.handler.DBPHandler is not a @Sharable handler, so can't be added or removed multiple times.
	at io.netty.channel.DefaultChannelPipeline.checkMultiplicity(DefaultChannelPipeline.java:449)
	at io.netty.channel.DefaultChannelPipeline.addLast0(DefaultChannelPipeline.java:146)
	at io.netty.channel.DefaultChannelPipeline.addLast(DefaultChannelPipeline.java:141)
	at io.netty.channel.DefaultChannelPipeline.addLast(DefaultChannelPipeline.java:253)
	at io.netty.channel.DefaultChannelPipeline.addLast(DefaultChannelPipeline.java:240)
	at com.tiza.util.client.impl.DBPClient$1.initChannel(DBPClient.java:84)
	at com.tiza.util.client.impl.DBPClient$1.initChannel(DBPClient.java:76)
	at io.netty.channel.ChannelInitializer.channelRegistered(ChannelInitializer.java:68)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRegistered(AbstractChannelHandlerContext.java:133)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRegistered(AbstractChannelHandlerContext.java:119)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRegistered(DefaultChannelPipeline.java:725)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:450)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.access$100(AbstractChannel.java:378)
	at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:424)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:358)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:357)
	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:112)
	at io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:137)
	at java.lang.Thread.run(Thread.java:745)
[ WARN ] [2016-03-23 12:21:57] io.netty.channel.ChannelInitializer.nioEventLoopGroup-9-1 [151] - Failed to initialize a channel. Closing: [id: 0x7a0a62ca]
io.netty.channel.ChannelPipelineException: com.tiza.util.client.handler.DBPHandler is not a @Sharable handler, so can't be added or removed multiple times.
	at io.netty.channel.DefaultChannelPipeline.checkMultiplicity(DefaultChannelPipeline.java:449)
	at io.netty.channel.DefaultChannelPipeline.addLast0(DefaultChannelPipeline.java:146)
	at io.netty.channel.DefaultChannelPipeline.addLast(DefaultChannelPipeline.java:141)
	at io.netty.channel.DefaultChannelPipeline.addLast(DefaultChannelPipeline.java:253)
	at io.netty.channel.DefaultChannelPipeline.addLast(DefaultChannelPipeline.java:240)
	at com.tiza.util.client.impl.DBPClient$1.initChannel(DBPClient.java:84)
	at com.tiza.util.client.impl.DBPClient$1.initChannel(DBPClient.java:76)
	at io.netty.channel.ChannelInitializer.channelRegistered(ChannelInitializer.java:68)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRegistered(AbstractChannelHandlerContext.java:133)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRegistered(AbstractChannelHandlerContext.java:119)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRegistered(DefaultChannelPipeline.java:725)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:450)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.access$100(AbstractChannel.java:378)
	at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:424)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:358)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:357)
	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:112)
	at io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:137)
	at java.lang.Thread.run(Thread.java:745)
[ WARN ] [2016-03-23 12:22:07] io.netty.channel.ChannelInitializer.nioEventLoopGroup-10-1 [151] - Failed to initialize a channel. Closing: [id: 0xfb2e28cc]
io.netty.channel.ChannelPipelineException: com.tiza.util.client.handler.DBPHandler is not a @Sharable handler, so can't be added or removed multiple times.
	at io.netty.channel.DefaultChannelPipeline.checkMultiplicity(DefaultChannelPipeline.java:449)
	at io.netty.channel.DefaultChannelPipeline.addLast0(DefaultChannelPipeline.java:146)
	at io.netty.channel.DefaultChannelPipeline.addLast(DefaultChannelPipeline.java:141)
	at io.netty.channel.DefaultChannelPipeline.addLast(DefaultChannelPipeline.java:253)
	at io.netty.channel.DefaultChannelPipeline.addLast(DefaultChannelPipeline.java:240)
	at com.tiza.util.client.impl.DBPClient$1.initChannel(DBPClient.java:84)
	at com.tiza.util.client.impl.DBPClient$1.initChannel(DBPClient.java:76)
	at io.netty.channel.ChannelInitializer.channelRegistered(ChannelInitializer.java:68)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRegistered(AbstractChannelHandlerContext.java:133)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRegistered(AbstractChannelHandlerContext.java:119)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRegistered(DefaultChannelPipeline.java:725)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:450)
	at io.netty.channel.AbstractChannel$AbstractUnsafe.access$100(AbstractChannel.java:378)
	at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:424)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:358)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:357)
	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:112)
	at io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:137)
	at java.lang.Thread.run(Thread.java:745)
[ WARN ] [2016-03-23 12:34:22] com.tiza.util.client.impl.DBPClient.Thread-2 [100] - DBP客户端重连失败！
